lib/endpoints/boot_params.js

```diff
@@ -265,7 +319,24 @@ BootParams.getByUuid = function handlerBootParamsGetByUuid(req, res, next) {
  * @section Boot Parameters API
  *
  * @param {Object} kernel_args Boot parms to update
- * @param {Array} boot_modules List of boot module objects
+ * @param {Array} boot_modules List of boot module objects.
```

Comment:
We need to update the "Default" endpoints too I think?

Reply:
It wasn't the original idea but, given there isn't too much work there, I'm not opposed.